scrapi/harvesters/lwbin.py

```diff
@@ -0,0 +1,210 @@
+    """
+    words = name.split(' ')
+    return any(word.strip(";") in ORGANIZATIONS for word in words)
+
+
+def process_contributors(authors, emails):
+    """Process authors and add author emails
+    If multiple authors and one email, put email in a new author
+    """
+
+    authors = authors.strip().replace('<span class="author-names">', '').replace('</span>', '')
+    authors = authors.split(',')
+    emails = emails.split(',')
+
+    for author in authors:
+        if is_organization(author):
+            break
+    else:
+        if ' and ' in authors[-1] or ' <em>et al.</em>' in authors[-1]:
+            split_name = authors.pop(-1).replace(' <em>et al.</em>', '').split(' and ')
+            authors.extend(split_name)
+
+    contributor_list = []
+    append_emails = True
+    if len(authors) != 1 or len(emails) != 1 or emails[0] == u'':
+        append_emails = False  # append the email to the author only when 1 record is observed
+
+    for ind, author in enumerate(authors):
+        if is_organization(author):
+            contributor = {
+                'name': author,
+                'sameAs': []
+            }
+        else:
+            name = HumanName(author)
+            contributor = {
+                'name': author,
+                'givenName': name.first,
+                'additionalName': name.middle,
+                'familyName': name.last,
+                'sameAs': []
+            }
+        if append_emails:
+            contributor['email'] = emails[ind]
+        contributor_list.append(contributor)
+
+    if not append_emails and emails[0] != u'':
+        for email in emails:
+            contributor = {
+                'name': '',
+                'givenName': '',
+                'additionalName': '',
+                'familyName': '',
+                'email': email,
+                'sameAs': [],
+            }
+            contributor_list.append(contributor)
+
+    return contributor_list
+
+
+def process_licenses(license_title, license_url, license_id):
+    """Process licenses to comply with the noormalized schema
+    """
+
+    if not license_url:
+        return []
+    else:
+        license = {
+            'uri': license_url,
+            'description': "{} ({})".format(license_title, license_id) or ""
+        }
+
+        return [license]
+
+
+def construct_url(url, dataset_path, end_point):
+    """Return a url that directs back to the page on LBWIN Data Hub instead of the source page.
+
+    Keyword arguments:
+    url -- host url
+    dataset_path -- parent path of all datasets
+    end_point -- name of datasets
+    """
+
+    return "/".join([url, dataset_path, end_point])
+
+
+def process_object_uris(url, extras):
+    """Extract doi from /extras, and return a list or object uris including /url and doi if it exists.
+    """
+    doi = ""
+    for d in extras:
+        if d['key'] == "DOI" or d['key'] == "DOI:":
+            doi = d['value']
+            break
```

Comment:
@fabianvf I don't see any. I can make it compatible to more than one dois though.